asyncy/omg/ServiceContract.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from ..Exceptions import AsyncyError, OmgPropertyTypeMissingError, \
+    OmgPropertyValueMismatchError
+
+
+class ServiceContract:
+
+    @classmethod
+    def check_value_with_type(cls, typ, value):
+        """
+        Validates for types listed on
+        https://microservice.guide/schema/actions/#arguments.
+
+        Supported types: int, float, string, list, map, boolean, or any
+        """
+        if typ is None:
+            raise OmgPropertyTypeMissingError(typ)
+        if typ == 'string':
+            cls.check_types(str, value)
+        if typ == 'int':
+            cls.check_types(int, value)
+        if typ == 'boolean':
+            cls.check_types(bool, value)
+        if typ == 'map':
+            cls.check_types(dict, value)
+        if typ == 'float':
+            cls.check_types(float, value)
+        if typ == 'list':
+            cls.check_types(list, value)
+        if typ == 'any':
+            return
+
+    @classmethod
+    def validate_output_properties(cls, output: dict, body, story,
+                                   line):
+        """
+        Verify all properties are contained in the return body
+        """
+        # if body is None or {}:
+        #    return
+        if output is None or {}:
+            return
+        props = output.get('properties')
+        for key in props:
```

Comment:
```suggestion
        for key, value in props.items():
```

You save a lot of `props.get(key)` then.

Reply:
done